apps/web-giddh/src/app/settings/customer-portal/customer.portal.component.ts

```
@@ -732,4 +783,218 @@ export class CustomerPortalComponent implements OnInit, AfterViewInit {
+        }
+    }
+
+    /**
+    * Search query change handler
+    *
+    * @param {string} query Search query
+    * @param {number} [page=1] Page to request
+    * @param {Function} successCallback Callback to carry out further operation
+    * @memberof CustomerPortalComponent
+    */
+    public payuOnAccountSearchQueryChanged(query: string = "", page: number = 1, successCallback?: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)